components/camel-kafka/src/main/java/org/apache/camel/component/kafka/KafkaConfiguration.java

```diff
@@ -5,9 +5,9 @@
  * The ASF licenses this file to You under the Apache License, Version 2.0
  * (the "License"); you may not use this file except in compliance with
  * the License.  You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
```

Comment:
don't change the license header, what we expect is https://github.com/apache/camel/blob/master/etc/apache-header.txt